apps/sms/views/conversation/js/conversation.js

```diff
@@ -1197,14 +1197,23 @@ var ConversationView = {
     var navigateToComposer = () => {
       var draftCreatePromise;
 
+      var draft;
       if (parameters.messageId) {
         draftCreatePromise = this.storeDraftFromMessage(parameters.messageId);
       } else if (parameters.number) {
-        var draft = new Draft({
+        draft = new Draft({
           recipients: [parameters.number],
           type: Utils.isEmailAddress(parameters.number) ? 'mms' : 'sms'
         });
         draftCreatePromise = Drafts.add(draft).store().then(() => draft.id);
+      } else if (parameters.draft) {
```

Comment:
nit: please augment the jsdoc for initiateNewMessage for the new parameter (`content` if you follow my suggestion)